### DISCUSSION TITLE
Cannot install versions newer than 3.3.0 via python api

### DISCUSSION THREAD

**Discussion by galvorbak99** [alphaparrot/ExoPlaSim#38] [Q&A] [answered]
I have been trying to update to version 3.4.0, and when I use `pip install exoplasim` it installs 3.3.0. When i try to manually specify the version, it gives me a "cannot find repository" message. Is there a way to install the newer versions via python, or do I have to download and compile manually?

**Comment by alphaparrot** [comment, marked as answer]
Hi, you're likely using too old a version of Python. The transition to 3.40
sounds like when I added support for the newest versions of Python, numpy,
scipy, etc and had to drop support for old versions of Python. You now must
have Python 3.8 or newer.

I recognize this is not ideal, as scientific computing environment
lifetimes are measured in units of PhDs, but newer versions of numpy and
Python had breaking changes that I unfortunately could not support without
dropping support for older python environments (including a Python 2.7
environment I had myself been using for research).

**Reply by galvorbak99** [reply]
Thank you for the timely answer! luckily this is all on my personal machine so it should be a quick fix

**Comment by alphaparrot** [comment]
If you're on unix (mac or linux) and have been using the system python
(which may be a pain to upgrade), ExoPlaSim generally plays nice with
Anaconda (and OpenMPI can usually be installed without undue difficulty via
the conda-forge channel).
